.github/workflows/functional_test_v2.yaml

```diff
@@ -224,6 +224,78 @@ jobs:
           cd functional_tests
           go test -v -tags functional
 
+  eks-upgrade-from-release-test:
+    name: Test helm upgrade from release in EKS - credentials needed
+    needs: kubernetes-test
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    concurrency:
+      group: eks-access
+    env:
+      KUBECONFIG: /tmp/kube-config-splunk-otel-collector-chart-functional-testing-eks
+      KUBE_TEST_ENV: eks
+      SKIP_TESTS: "true" # we need to skip functional tests as we have not set probes to listen to the traffic.
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Checkout main if in a PR
+        if: github.event_name == 'pull_request'
+        uses: actions/checkout@v4
+        with:
+          path: base
+          ref: main
+      - name: Checkout previous main commit if in main
+        if: github.event_name != 'pull_request'
+        uses: actions/checkout@v4
+        with:
+          path: base
+          ref: HEAD
+          fetch-depth: 2
+      - name: Checkout fixed previous release v0.109.0
+        if: github.event_name != 'pull_request'
+        run: cd base && git checkout splunk-otel-collector-0.109.0
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ~1.21.8
+          cache: false
+      - name: Cache Go
+        id: go-cache
+        timeout-minutes: 5
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/go/bin
+            ~/go/pkg/mod
+          key: go-cache-${{ runner.os }}-${{ hashFiles('**/go.sum') }}
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET }}
+          aws-region: us-west-1
+      - name: Install kubeconfig
+        run: |
+          aws eks update-kubeconfig --name rotel-eks --region us-west-1
+      - name: Update dependencies
+        run: |
+          cd base && make dep-update
+      - name: Deploy cert-manager
+        run: |
+          cd base && make cert-manager
+      - name: Deploy previous version of the chart
+        run: |
+          cd base && helm install sock helm-charts/splunk-otel-collector --set cloudProvider=aws --set distribution=eks --set splunkObservability.realm=us0 --set splunkObservability.accessToken=xxxxx
```

Comment:
```suggestion
          cd base && helm install sock helm-charts/splunk-otel-collector --set cloudProvider=aws --set distribution=eks --set splunkObservability.realm=us0 --set splunkObservability.accessToken=xxxxx --set operator.enabled=true --set certmanager.enabled=true
```

Reply:
The default has operator disabled in last version https://github.com/signalfx/splunk-otel-collector-chart/blob/splunk-otel-collector-0.113.0/helm-charts/splunk-otel-collector/values.yaml#L1168. If the goal is to test for operator/CRD changes are no-op for customers running with operator, then we should the extra values ^